doc/api/sqlite.md

```diff
@@ -156,6 +156,10 @@ around [`sqlite3_prepare_v2()`][].
 
 ### `database.createSession([options])`
 
+<!-- YAML
+added: v23.0.0
```

Comment:
https://github.com/nodejs/node/pull/54181 was added in [v23.3.0](https://github.com/nodejs/node/pull/55921).

Reply:
Oops, thanks!